libva.rb

```diff
@@ -20,14 +16,15 @@ class Libva < Formula
   option "with-eglx", "Build libva with egl and glx support (use after building mesa)"
 
   # Build-time
-  depends_on "pkg-config" => :build
   depends_on "autoconf" => :build
-
+  depends_on "pkg-config" => :build
+  depends_on "libtool" => :build if build.without?("wayland")
   depends_on "linuxbrew/xorg/libdrm"
+  depends_on "linuxbrew/xorg/libx11"
+  depends_on "linuxbrew/xorg/libxext"
+  depends_on "linuxbrew/xorg/libxfixes"
```

Comment:
My understanding is that `libva` will build without X11 if those libraries aren't present.

Since it isn't a part of core X11, might it make sense for `libva` to just depend on `:x11` rather than specific libraries? We can also make it `:recommended` if you think that'd make more sense; I'm no expert on it.